event_sale_registration_multi_qty/wizards/event_edit_registration.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+<!-- Copyright 2017 Tecnativa - Sergio Teruel
```

Comment:
Here should cover year 2017-2019. @chienandalu apologies if asking again but maybe GitHub has some issues to keep my comments